src/OrchardCore/OrchardCore.Mvc.Core/Extensions/ControllerBaseExtensions.cs

```diff
@@ -25,15 +25,15 @@ public static ActionResult ChallengeOrForbid(this Controller controller)
         /// <param name="controller"></param>
         /// <param name="authenticationSchemes">The authentication schemes to challenge.</param>
         /// <returns>The proper actionresult based upon if the user is authenticated.</returns>
-        public static ActionResult ChallengeOrForbid(this Controller controller, params string[] authenticationSchemes)
+        public static ActionResult ChallengeOrForbid(this ControllerBase controller, params string[] authenticationSchemes)
```

Comment:
@sebastienros There is also this which needed to also be extended on top of the ControllerBase just because we need it.